app/models/listing.rb

```diff
@@ -2,7 +2,6 @@ class Listing < ApplicationRecord
   # We used to use both "classified listing" and "listing" throughout the app.
   # We standardized on the latter, but keeping the table name was easier.
   self.table_name = "classified_listings"
-  self.ignored_columns = ["last_buffered"].freeze
```

Comment:
Not related to this PR, but it's a leftover and needs to be removed, so I left it in.